### PR TITLE
home page ui tweaks

### PR DESCRIPTION
**Pull request by vojtaholik**
home-tweaks@2x

### CODE REVIEW

File: src/components/pages/home/homepage-data.ts [hunk 1]
```diff
@@ -27,7 +27,7 @@ const homepageData = [
 There are a number of tried and true methods that will allow you to learn more efficiently so that you can keep up with the industry. These evergreen methods will serve you well no matter your level of expertise.
       `,
     instructor: 'Ceora Ford',
-    instructor_path: '/q/resources-by-nader-dabit',
+    instructor_path: '/q/resources-by-ceora-ford',
```
Comment (vojtaholik) [inline src/components/pages/home/homepage-data.ts:30]: fixing instructor path

File: src/components/pages/home/homepage-data.ts [hunk 2]
```diff
@@ -214,12 +214,11 @@ There are a number of tried and true methods that will allow you to learn more e
         path: '/courses/practical-git-for-everyday-professional-use',
       },
       {
-        title: 'Develop Accessible Web Apps with React',
-        byline: 'Erin Doyle・88m・Course',
+        title: 'Build an App with React Suspense',
+        byline: 'Michael Chan・1h 23m・Course',
```
Comment (vojtaholik) [inline src/components/pages/home/homepage-data.ts:218]: we already have this course on the page so I switched it up for a different one

File: src/components/pages/home/card/index.tsx [hunk 3]
```diff
@@ -38,7 +38,7 @@ const Card: FunctionComponent<CardProps> = ({
 
   return (
     <div
-      className={`bg-white dark:bg-gray-800 dark:text-gray-200 shadow-sm rounded-lg overflow-hidden sm:p-5 p-4 ${
+      className={`bg-white dark:bg-gray-800 dark:text-gray-200 shadow-sm rounded-lg overflow-hidden sm:p-8 p-5 ${
```
Comment (vojtaholik) [inline src/components/pages/home/card/index.tsx:41]: larger paddings in cards

File: src/components/pages/home/index.tsx [hunk 4]
```diff
@@ -63,17 +63,17 @@ const Home: FunctionComponent = () => {
...
         </section>
-        <section className="grid lg:grid-cols-8 grid-cols-1 gap-5 ">
+        <section className="grid lg:grid-cols-8 grid-cols-1 lg:gap-6 gap-4">
           <FeaturedVideoCard video={video} />
           <EventSchedule />
         </section>
-        <section className="grid lg:grid-cols-12 grid-cols-1 gap-5 ">
-          <div className="lg:col-span-8 space-y-8">
+        <section className="grid lg:grid-cols-12 grid-cols-1 lg:gap-6 gap-4">
+          <div className="lg:col-span-8 lg:space-y-6 space-y-4">
```
Comment (vojtaholik) [inline src/components/pages/home/index.tsx:76]: consistent spacing between cards

File: src/components/pages/home/homepage-data.ts (hunk 2 above)
Comment (laurosilvacom) [reply src/components/pages/home/homepage-data.ts:218]: This makes sense!

**Comment by zacjones93** [comment]
I like the consistent spacing

File: src/components/pages/home/card/index.tsx (hunk 3 above)
Comment (joelhooks) [reply src/components/pages/home/card/index.tsx:41]: I think the larger gaps looked better here, but not enough to really take a stand ;)

**Review by joelhooks — APPROVED** [review]

**Pull request merged**